commerce_coordinator/apps/commercetools/catalog_info/edx_utils.py

```diff
@@ -40,6 +40,29 @@ def is_edx_lms_order(order: CTOrder) -> bool:
     return len(get_edx_items(order)) >= 1
 
 
+def get_line_item_attribute(in_line_item, in_attribute_name):  # pragma no cover
+    """Utility to get line item's attribute's value."""
+    attribute_value = None
+    for attribute in in_line_item.variant.attributes:
+        if attribute.name == in_attribute_name and hasattr(attribute, 'value'):
+            if isinstance(attribute.value, dict):
+                attribute_value = attribute.value.get('label', None)
+            elif isinstance(attribute.value, str):
+                attribute_value = attribute.value
+            break
+
+    return attribute_value
+
+
+def get_course_mode_from_ct_order(line_item: CTLineItem) -> str:
+    course_mode = get_line_item_attribute(line_item, 'mode')
+    mode = 'verified'
+    if course_mode is not None:
+        mode = course_mode
+
+    return mode
```

Comment:
return course_mode or 'verified'

Reply:
Sorry I did not understand I am returning the mode either 'verified' or actual course_mode

Reply:
These four lines can be turned to a single line of code which does the same thing. If course_mode exists we return that or else we return our hard coded value.